ash/Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "ash"
-version = "0.37.0+1.3.235"
-authors = ["maik klein <maikklein@googlemail.com>"]
+version = "0.37.1+1.3.235"
+authors = [
+    "Maik Klein <maikklein@googlemail.com>",
+    "Benjamin Saunders",
+    "Marijn Suijten <marijn@traverseresearch.nl>",
+]
```

Comment:
@Ralith mind if I extend the authors some, like this? :grimacing:

Reply:
Please do! Might as well include ` <ben.e.saunders@gmail.com>` too.

Reply:
Gotcha!